src/libsystemd/sd-json/json-util.h

```diff
@@ -153,6 +154,7 @@ enum {
         _JSON_BUILD_PAIR_IN_ADDR_NON_NULL,
         _JSON_BUILD_PAIR_ETHER_ADDR_NON_NULL,
         _JSON_BUILD_PAIR_HW_ADDR_NON_NULL,
+        _JSON_BUILD_PAIR_RATELIMIT_NON_NULL,
```

Comment:
'null ratelimit' sounds spurious for me. Maybe, _RATELIMIT_ENABLED or _ENABLED_RATELIMIT?